src/components/AppBar/components/HamburgerButton/HamburgerButton.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import { hideVisually } from 'polished';
+import HamburgerIcon from './hamburger.svg';
+
+const baseStyles = ({ theme }) => css`
+  label: appbar__hamburguer-button;
+  outline: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 32px;
+  width: 32px;
+  border: ${theme.borderWidth.kilo} solid ${theme.colors.n700};
+  border-radius: 50%;
+  background-color: ${theme.colors.n900};
+  cursor: pointer;
+`;
+
+const AccessibilityLabel = styled.span`
+  ${hideVisually()};
+`;
+
+const Button = styled('button')(baseStyles);
+
+const HamburgerButton = ({ hamburgerButtonLabel, onClick }) => (
+  <Button onClick={onClick}>
+    <AccessibilityLabel data-selector="accessibility-label">
+      {hamburgerButtonLabel}
+    </AccessibilityLabel>
+    <HamburgerIcon />
```

Comment:
Not 100% sure, but wouldn't you put some special accessibility attributes on SVGs? I think we tried doing this in the [payment icons](https://github.com/sumup/circuit-ui/blob/master/src/components/CreditCardDetails/components/scheme-icons/card-scheme-icons.js#L58). The role prevents screen readers from reading out the SVGs markup, if I remember correctly. And I guess you can save yourself the extra label component by using the `aria-label` attribute?

Reply:
Good point @felixjung! Although a label element has better support from screen reader tools than the `aria-label` attribute.